apps/coscrad-frontend-e2e/src/e2e/resource-index-to-detail-flows/spatial-feature.query-flow.e2e.cy.ts

```diff
@@ -0,0 +1,60 @@
+describe(`Spatial Feature index-to-detail flow`, () => {
+    const textForSpatialFeatureMarker = 'Name of Point with ID: 103';
```

Comment:
Be more specific if you have more than one spatial feature. Maybe textForPoint103? or `textForPointToExpand`?